README.md

```diff
@@ -1,14 +1,15 @@
 # cgpVAF
 
+| Master                                        | Develop                                         |
+| --------------------------------------------- | ----------------------------------------------- |
+| [![Master Badge][travis-master]][travis-base] | [![Develop Badge][travis-develop]][travis-base] |
+
+
 Calculates the Variant Allele Fraction for variants sites in VCF and/or BED file 
 This script performs comparative analysis of variant sites in multiple tumour/normal samples in an individual.
 Also facilitates the merging of varinats sites across the samples in a sample group defined by 
 set of related samples in an individual and provides unbiased pileup[MNV] and exonerate[Indel] output for each variant site.
 
-| Master                                        | Develop                                         |
-| --------------------------------------------- | ----------------------------------------------- |
-| [![Master Badge][travis-master]][travis-base] | [![Develop Badge][travis-develop]][travis-base] |
-
```

Comment:
You need to add the links `[travis-***]` is a defined url at the very bottom of the example I sent you

Reply:
Thanks, done now